extensions/amp-list/validator-amp-list.protoascii

```diff
@@ -144,9 +144,39 @@ tags: { # amp-list-load-more
     name: "load-more-end"
     value: ""
     mandatory_oneof: "['load-more-button', 'load-more-failed', 'load-more-end', 'load-more-loading']"
-  }
+  },
 }
 
+# The button element variant allowed in amp-list-load-more.
+tags: {
+  html_format: AMP
+  html_format: AMP4ADS
+  html_format: AMP4EMAIL
+  html_format: ACTIONS
+  requires_extension: "amp-list"
+  tag_name: "BUTTON"
+  spec_name: "amp-list-load-more button[load-more-clickable]"
+  mandatory_ancestor: "AMP-LIST-LOAD-MORE"
```

Comment:
Can this be a parent requirement, instead of an ancestor? Better to be more specific and affects dispatch_key, see later comment.

Reply:
done!